src/main/java/org/agileandbeyond/dandd/tddexercise/Race.java

```diff
@@ -0,0 +1,147 @@
+package org.agileandbeyond.dandd.tddexercise;
+
+public class Race {
```

Comment:
I would really prefer to see the Race class as an abstract class and have actual concrete Races extend the Race class.  So, the Human Race class would hide the logic to set the RaceType to human.  This is a very common pattern in Object Oriented design.  You can code it like you have done with the consumer specifying the Race Type, but that is less Object Oriented.  Using this type pattern, you will find the code being very if-else, or in this case switch-case, heavy.  You are forced to put all Race specific code in the switches that are defined for that particular race instead of putting it all neatly in the concrete race class.